docs/Reference/Console/configuration.md

```diff
@@ -0,0 +1,38 @@
+# Console Configuration
+
+Console can be configured using either a YAML configuration file (`config.yml`) or environment variables (`.env`). Both methods allow you to customize the behavior of the Console application.
+
+## Using config.yml
+
+When you first run the Console BETA, a `config.yml` file is generated and stored in the `config/` directory, located in the same directory as the Console executable. You can stop the Console, edit the config.yml file, and then restart the Console to apply the changes.
+
+Alternatively, users can also download the [config.yml file from GitHub](https://github.com/open-amt-cloud-toolkit/console/blob/main/config/config.yml) or create their own config.yml file, edit the YAML content, and then use the config flag to run the Console ```console.exe -config config.yaml```
+
+## Using .env Variables
+
+Console can also be configured using environment variables. These `.env` variables take precedence over the corresponding settings in the `config.yml` file, effectively overwriting them.
+
+## Console configuration Variables
+
+| `.env` Variable Name | `config.yml` Variable Name       | Default | Description |
```

Comment:
config.yml Variable names are missing the hierarchy. example name, repo, port, etc.. should probably be `app:name` to indicate they are nested under the app section. Additionally, these ENV names don't look correct, none of them begin with CONSOLE_? they should be APP_. It might not be a bad idea to add a console prefix, but still need the APP which corresponds to the section 
![image](https://github.com/user-attachments/assets/3ab7c5d9-cb95-4ca0-b488-bf2944df76b2)

Reply:
Missed this but should be resolved now. Made sure that the variables match, used app.name instead of app:name.